llvm/utils/lit/tests/Inputs/shtest-env/env-args-none.txt

```diff
@@ -1 +1,30 @@
-# RUN: env
+# Check Default Environment
+# RUN: env | FileCheck -check-prefix=CHECK-2-EMPTY %s
```

Comment:
Lets update the prefix here. In the example you drew from there were 2 env uses, so we don't need the `2`. I'd also say you don't need the `CHECK- ` prefix. These comments should apply to all the prefixes in the file.

So, something like the following seems a bit better.
```suggestion
# RUN: env | FileCheck -check-prefix=EMPTY %s
```

Reply:
Please don't use any check prefix ending in `EMPTY`, because FileCheck has an `EMPTY` suffix for checking if a line is completely empty.